doc/source/project/organization.rst

```diff
@@ -106,12 +106,13 @@ Pull requests changing or clarifying the procedures governing this repository:
   *-1* binding votes.
 - Should be titled with the prefix *[PROCEDURES]* and tagged with
   the *procedures* tag in Github.
-- Should not be modified once open. If changes are needed, the pull request
-  should be closed, re-opened with modifications, and votes reset.
 - Should be restricted to just modifying the procedures and generally should not
   contain code modifications.
 - If the pull request adds or removes *committers*, there must be a separate
   pull request for each person added or removed.
+- Amendments may be made to the pull request by pushing additional changesets
+  but doing so restarts the moratorium and invalidates any votes which have
```

Comment:
Well, I explicitly wanted to require anyone who'd made a -1 to affirm a +1 (or otherwise retract their -1) prior to merge.  That seem right?